gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1021,39 +1021,37 @@ public Builder sourceOptions(Iterable<BlobSourceOption> options) {
        *
        * @return the builder
        */
-      public Builder target(BlobId target) {
-        this.target = BlobInfo.builder(target).build();
+      public Builder target(BlobId targetId) {
+        this.targetId = targetId;
         return this;
       }
 
       /**
        * Sets the copy target and target options. {@code target} parameter is used to override
-       * source blob information (e.g. {@code contentType}, {@code contentLanguage}). {@code
-       * target.contentType} is a required field.
+       * source blob information (e.g. {@code contentType}, {@code contentLanguage}). Target blob
+       * information is set exactly to {@code target}, no information is inherited from the source
+       * blob. If not set, target blob information is inherited from the source blob.
```

Comment:
I am not sure I understand the last statement, "f not set, target blob information is inherited from the source blob.", in this context.